components/calendar/Cargo.toml

```diff
@@ -37,10 +37,12 @@ denylist = ["bench"]
 
 [dependencies]
 displaydoc = { version = "0.2.3", default-features = false }
-tinystr = { path = "../../utils/tinystr", version = "0.5.0", features = ["alloc", "serde"], default-features = false }
+tinystr = { path = "../../utils/tinystr", version = "0.5.0", features = ["alloc", "zerovec"], default-features = false }
 icu_provider = { version = "0.5", path = "../../provider/core", features = ["macros"] }
 litemap = { version = "0.3.0", path = "../../utils/litemap", default-features = false }
 serde = { version = "1.0", default-features = false, features = ["derive", "alloc"], optional = true }
+zerovec = { version = "0.6", path = "../../utils/zerovec", default-features = false }
+zerovec-derive = { version = "0.6", path = "../../utils/zerovec/derive", default-features = false }
```

Comment:
we shouldn't be directly depending on derive crates 99% of the time

Perhaps wait for #1632?